src/Symfony/Component/DependencyInjection/Attribute/Locator.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\DependencyInjection\Attribute;
+
+use Symfony\Component\DependencyInjection\Argument\ServiceLocatorArgument;
+use Symfony\Component\DependencyInjection\ContainerInterface;
+use Symfony\Component\DependencyInjection\Reference;
+
+#[\Attribute(\Attribute::TARGET_PARAMETER)]
+final class Locator extends Autowire
+{
+    public function __construct(string ...$serviceIds)
+    {
+        $values = [];
+
+        foreach ($serviceIds as $key => $serviceId) {
+            $nullable = str_starts_with($serviceId, '?');
+            $serviceId = ltrim($serviceId, '?');
+
+            if (is_numeric($key)) {
+                $key = $serviceId;
+            }
+
+            $values[$key] = new Reference(
+                $serviceId,
+                $nullable ? ContainerInterface::IGNORE_ON_INVALID_REFERENCE : ContainerInterface::EXCEPTION_ON_INVALID_REFERENCE,
```

Comment:
`IGNORE_ON_INVALID_REFERENCE` is what we want here, correct?

Reply:
I guess so, this is tested, isn't it?

Reply:
Yes, I wanted to confirm we want `$container->get('nullable.service')` to _not_ return `null` (which would be the case if `NULL_ON_INVALID_REFERENCE` is used).